### DISCUSSION TITLE
Open date picker at last month used

### DISCUSSION THREAD

**Discussion by thepurpleblob** [directus/directus#11840] [Feature Requests]
The current behaviour seems to be always to open at the current month. If entering a long list of dates it's more likely that you will be working through different months. It would be more convenient to open at the month most recently selected.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
